frontend/src/lib/v2/WorkflowUtils.test.ts

```diff
@@ -17,9 +17,15 @@ import { Workflow, WorkflowSpec, WorkflowStatus } from 'third_party/argo-ui/argo
 import { getContainer, isV2Pipeline } from './WorkflowUtils';
 import { ComponentSpec } from 'src/generated/pipeline_spec';
 import fs from 'fs';
+import jsyaml from 'js-yaml';
 
-const V2_PIPELINESPEC_PATH = 'src/data/test/lightweight_python_functions_v2_pipeline_rev.yaml';
-const v2YamlTemplateString = fs.readFileSync(V2_PIPELINESPEC_PATH, 'utf8');
+const V2_LW_PIPELINESPEC_PATH = 'src/data/test/lightweight_python_functions_v2_pipeline_rev.yaml';
+const v2LWYamlTemplateString = fs.readFileSync(V2_LW_PIPELINESPEC_PATH, 'utf8');
+const V2_PVC_PIPELINESPEC_PATH = 'src/data/test/create_mount_delete_dynamic_pvc.yaml';
+const v2PVCYamlString = fs.readFileSync(V2_PVC_PIPELINESPEC_PATH, 'utf8');
```

Comment:
We should be consistent on the naming of const: It should be all capital in camel style.